pages/inspector/connect-inspector-to-gtm.mdx

```diff
@@ -1,14 +1,74 @@
+
+## Inspector Client side GTM integration (beta)
+
+There are two options to install Inspector via Google Tag Manager. You can either import our GTM recipe to add observability to all data layer events, or use our Inspector function to add observability to individual events.
+
+<Callout type="info" emoji="💡">
+We recommend starting by installing the Inspector on your development or testing GTM container and connect it to the Inspector dev environment. That is both so your Inspector production environment is not contaminated by testing data and because the Inspector development environment is designed to handle low data volumes real time while the production environment is optimized for large amount of production data and has 1 - 2 hour delay.
+</Callout>
+
+### Importable GTM recipe
+
+To add observability to all data layer events with our importable GTM recipe [in under 90 seconds](https://www.loom.com/share/c659b857d4d947729683bf11691f441e), follow the steps below:
+1. [Download our GTM recipe here]()
+2. Import the recipe into your GTM container. 
+3. Add the source specific Inspector API key. 
+4. When testing, adjust the the Avo Inspector Init tag by changing the 'prod' value to 'dev' on the inspector.__ENV__ key.
+5. Publish the container.
+6. Visit the Avo Inspector dashboard in one hour to see
+7. Consider creating a filtered issues view for the data bugs you care about. [learn more here](/inspector/inspector-issues-view).
+
+<Callout type="info" emoji="💡">
+The Initialization code is already included in a tag, so you just need to provide the Inspector API GTM variable the source specific API key, which you can find in your Avo source Inspector setup. 
+</Callout>
+
+### Inspector Get Schema function
+
+Another option is to initialize Inspector via creating your own HTML tag and using the HTML script you can find in the source Inspector setup.
+
+To add observability to single events with our Inspector Get Schema function, create a HTML script tag for each event you want to add observability to. The steps are shown in the screen shot below and make sure to use the same trigger and variables as you are using for the event you're dispatching to your analytics vendor. 
+
+<Callout type="info" emoji="💡">
+If you have occasions where data layer events could fire before GTM loads and you wish you add observability, we'd recommend initializing inspector outside of Google Tag Manager, and perhaps adding our initialization code to the header of the page or using one of our other Inspector SDKs and adding it more native to your codebase. The flexibility here is a feature.
```

Comment:
@thoragudf typo. 

Change "fire before GTM loads and you wish you add observability,"

to 

fire before GTM loads and you wish to add observability,